test/test_processing_utils.py

```diff
@@ -421,7 +421,7 @@ async def test_json_data_not_moved_to_cache():
 )
 def test_local_urls_fail(url):
     with pytest.raises(httpx.RequestError, match="Non-public IP address found"):
```

Comment:
Looks like you'll need to update these tests to just check if `get_public_url` returns `None` instead since we no longer raise errors in the method